apps/frontend/src/renderer/components/Sidebar.tsx

```
@@ -298,23 +389,23 @@ export function Sidebar({
   return (
     <TooltipProvider>
       <div className="flex h-full w-64 flex-col bg-sidebar border-r border-border">
-        {/* Header with drag area - extra top padding for macOS traffic lights */}
-        <div className="electron-drag flex h-14 items-center px-4 pt-6">
-          <span className="electron-no-drag text-lg font-bold text-primary">Auto Claude</span>
+        {/* Header with drag area */}
+        <div className="electron-drag flex h-12 items-center px-4 shrink-0 border-b border-border bg-sidebar/95 backdrop-blur">
+          <span
+            className="electron-no-drag text-sm font-bold text-primary tracking-tight cursor-default"
+            onDoubleClick={() => window.electronAPI.maximizeWindow()}
+          >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)